project/urls.py

```
@@ -23,6 +23,7 @@
     path('admin/', admin.site.urls),
     path('', include('home.urls')),
     path('accounts/', include('accounts.urls')),
+    path('classroom/',include('classrooms.urls'))
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)